Update Environment Variables link on README

Adarsh-NP opened the pull request:
Changed the link assosciated with environment variables from wiki to the repo link as mentioned in the issue.

### Description
Earlier the link explaining the exporting of environment variables was to a wiki page which asked to refer to mentorship-backend/docs/environment-variables.md. I simply connected the link directly to the file thus skipping the wiki link.

Fixes #494 

### Type of Change:
- Documentation



### How Has This Been Tested?
The new link takes to the appropriate directory.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)

Adarsh-NP committed:
Updated rREADME.md

changed the link assosciated with environment variables from wiki to the repo link as mentioned in the issue.

isabelcosta commented on README.md:
```diff
@@ -13,7 +13,7 @@ This is the Backend REST API for the Mentorship System.
 
  - [Fork, Clone and Remote](https://github.com/anitab-org/mentorship-backend/wiki/Fork%2C-Clone-%26-Remote)
- - [Export Environment Variables](https://github.com/anitab-org/mentorship-backend/wiki/Environment-Variables)
+ - [Export Environment Variables](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/environment-variables.md)
```
```suggestion
 - [Export Environment Variables](docs/environment-variables.md)
```
what about using relative path? I think this could work :)

isabelcosta commented on README.md:
```diff
@@ -30,7 +30,7 @@ The project runs on Python 3.
 4. Make sure you create `.env` using `.env.template` and update the values of corresponding environment variables
 or
-make sure you exported the following [environment variables](https://github.com/anitab-org/mentorship-backend/wiki/Environment-Variables):
+make sure you exported the following [environment variables](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/environment-variables.md):
```
```suggestion
make sure you exported the following [environment variables](docs/environment-variables.md):
```
relative path

isabelcosta commented on README.md:
```diff
@@ -60,7 +60,7 @@ export DB_NAME=<database_name>
 
 ### Run with docker
-1. Make sure you exported the following [environment variables](https://github.com/anitab-org/mentorship-backend/wiki/Environment-Variables) to '.env' file
+1. Make sure you exported the following [environment variables](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/environment-variables.md) to '.env' file
```
```suggestion
1. Make sure you exported the following [environment variables](docs/environment-variables.md) to '.env' file
```

isabelcosta reviewed (changes requested):
Thank you for your contribution @Adarsh-NP !
Could you use relative paths? This is so if you are on your fork or we use another branch in the future then this is not tightly coupled to `develop` branch. Makes sense?
Also could you please follow our [Commit Message Style Guide](https://github.com/anitab-org/mentorship-backend/wiki/Commit-Message-Style-Guide) as this is part of our project guidelines.

Adarsh-NP committed:
Updated Readme.md

docs: updated the link to environment variables from wiki page to more updated file in docs directory.

This solves the issue #494.

isabelcosta commented:
@Adarsh-NP why did you close this issue? Do you wish to stop working on this issue?

isabelcosta reviewed (approved):
@Adarsh-NP this is great, but can you squash your commits and follow our commit style guideline?
Let me know if you need any help with this

Adarsh-NP commented:
@isabelcosta yeah ma'am, regarding commit message guidelines what else do I need to change, I don't seem to understand it.

isabelcosta commented:
@Adarsh-NP your main git commit message is `Updated Readme.md`, when following our Commit Message Style Guideline, that you can find on our Wiki and the CONTRIBUTING.md, your message should be something like: `docs: update env variables link on readme`.

Are you able to squash and amend the commit message?

isabelcosta commented:
@Adarsh-NP your pull request was merged 🎉 thank you for your contribution 👏
